pkg/ebpf/events_processor.go

```diff
@@ -340,8 +345,42 @@ func (t *Tracee) processEvent(event *trace.Event) error {
 
 	case InitModuleEventID:
 		t.invokeIoctlTriggeredEvents()
+	case DetectHookedProcFopsEventID:
+		fopsAddresses, err := getEventArgUlongArrVal(event, "hooked_fops_pointers")
+		if err != nil || fopsAddresses == nil {
+			return fmt.Errorf("error parsing detect_hooked_proc_fops args: %w", err)
+		}
+		hookedFops := make([]bufferdecoder.HookedSymbolData, 0, 0)
+		for idx, addr := range fopsAddresses {
+			inTextSeg, err := t.kernelSymbols.TextSegmentContains(addr)
+			if err != nil {
+				return fmt.Errorf("error checking kernel address: %v", err)
+			}
+			if !inTextSeg {
+				hookingFunction, err := t.kernelSymbols.GetSymbolByAddr(addr)
+				if hookingFunction.Owner == "system" && err == nil {
+					continue
+				}
+				if err != nil {
+					hookingFunction.Owner = "hidden"
+				} else {
+					hookingFunction.Owner = strings.TrimPrefix(hookingFunction.Owner, "[")
```

Comment:
it feels to me that [ and ] should be removed in GetSymbolByAddr function, no ?

Reply:
yep , ill open PR on that in libbpfgo to change it

Reply:
Hi i, updating that ill tried to create a fix for that in libbpfgo, for some reason i couldn't run my libpfgo commit with the PR, I'd love to do it in the following days but I'm supposed to be offline for the couple weeks so i open the PR as draft there but ill be available for any Q in the git (: